docs/proposals/localDisk_volume.md

```diff
@@ -0,0 +1,309 @@
+# localDisk volume
+
+## Abstract
+
+This is a proposal for implementing a new volume type, localDisk, which will support,
+
+ 1. the Node has multiple physical disks and pod requires 1 ~ N disks as its storage volumes.
+ 2. the Node has lvm VG and pods allocate specific size LVs from this VG.
+ 3. When pods are deleted, the data in the volume could be either retained or deleted.
+ 
+As localDisk volume life cycle would be different from pod, we need volume APIs to do the management. 
+The existing PersistentVolume/PersistentVolumeClaim(PV/PVC) API is for network storage in cluster, we propose new LocalVolume/LocalVolumeClaim(LV/LVC) API to manage localDisk volume.                      
+
+## Motivation
+
+* This new volume type is for application that requires high I/O performance, one pod need a dedicated disk.

```

Comment:
Yes, using cgroup to guarantee QoS could be next step.
As first step, one disk for one pod will be more easy for I/O QoS guarantee, and easy implementation.